tests/spec/core/biblio-spec.js

```diff
@@ -14,37 +15,147 @@ describe("W3C — Bibliographic References", () => {
         href: "http://test.com",
         authors: ["Another author"],
-        publisher: "Testing 123",
+        publisher: "Testing 123"
       },
       TestRef3: {
         title: "Third test",
         href: "http://test.com",
-        publisher: "Publisher Here",
+        publisher: "Publisher Here"
       },
       FOOBARGLOP: {
-        aliasOf: "BARBAR",
+        aliasOf: "BARBAR"
       },
       BARBAR: {
-        title: "The BARBAR Spec",
+        title: "The BARBAR Spec"
+      }
+    }
+  };
+
+  //Configuration for APA citation
+  const conf2 = {
+    editors: [
+      {
+        name: "Robin Berjon"
+      }
+    ],
+    edDraftURI: "https://foo",
+    shortName: "Foo",
+    specStatus: "WD",
+    prevVersion: "FPWD",
+    previousMaturity: "WD",
+    previousPublishDate: "2013-12-17",
+    biblioStyle: "APA",
+    localBiblio: {
+      Zzz: {
+        title: "Last Reference"
       },
-    },
+      aaa: {
+        title: "First Reference"
+      },
+      TestRef4: {
+        title: "Test ref title",
+        href: "http://test.com",
+        authors: ["William Shakespeare"]
+      },
+      TestRef5: {
+        title: "Second test",
+        href: "http://test.com"
+      },
+      TestRef6: {
+        title: "Third test"
+      },
+      TestRef7: {
+        authors: ["William Shakespeare"]
+      },
+      TestRef8: {
+        href: "http://test.com"
+      },
+      TestRef9: {
+        href: "http://test.com",
+        authors: ["William Shakespeare"]
+      },
+      FOOBARGLOP: {
+        aliasOf: "BARBAR"
+      },
+      BARBAR: {
+        title: "The BARBAR Spec"
+      }
+    }
   };
-  const body = `
+
+  //Configuration for MLA citation
+  const conf3 = {
+    editors: [
+      {
+        name: "Robin Berjon"
+      }
+    ],
+    edDraftURI: "https://foo",
+    shortName: "Foo",
+    specStatus: "WD",
+    prevVersion: "FPWD",
+    previousMaturity: "WD",
+    previousPublishDate: "2013-12-17",
+    biblioStyle: "MLA",
+    localBiblio: {
+      Zzz: {
+        title: "Last Reference"
+      },
+      aaa: {
+        title: "First Reference"
+      },
+      TestRef10: {
+        title: "Test ref title",
+        href: "http://test.com",
+        authors: ["William Shakespeare"],
+        publisher: "Publishers Inc."
+      },
+      TestRef11: {
+        title: "Second test",
+        href: "http://test.com"
+      },
+      TestRef12: {
+        title: "Third test",
+        publisher: "Publisher Here"
+      },
+      TestRef13: {
+        title: "Test ref title",
+        href: "http://test.com",
+        authors: ["William Shakespeare"]
+      },
+      TestRef14: {
+        title: "Test ref title",
+        authors: ["William Shakespeare"]
+      },
+      TestRef15: {
+        authors: ["William Shakespeare"],
+        publisher: "Publishers Inc."
+      },
+      FOOBARGLOP: {
+        aliasOf: "BARBAR"
+      },
+      BARBAR: {
+        title: "The BARBAR Spec"
+      }
+    }
+  };
+
+  const body1 = `
```

Comment:
You should be able to use just 1 body, don't need 3. Each body can then have a different citation style, based on the config.

Reply:
Alright, but how do I manage the change in number of tests cases? Or should I reduce all. To 3 test cases?